Rapid7 appsec (#29134)
demisto/content: Packs/Rapid7AppSec/Integrations/Rapid7AppSec/Rapid7AppSec_description.md

### Basic Information
Supports AppSec 3.0.0 and above.

> When using XSOAR credentials insert your API key to the credential password. Otherwise, insert your API key into the relevant required parameters.

Reviewer: ```suggestion
When using Cortex XSOAR credentials, insert your API key as the credential password. Otherwise, insert your API key into the relevant required parameters.
```

### Base URL
Fill the {region-code} by your region.

> United States: use region-code=us or us2 or us3.
> Europe: use region-code=eu.
> Canada: use region-code=ca.
> Australia: use region-code=au.
> Japan: use region-code=ap.

Reviewer: ```suggestion
- United States: use region-code=us or us2 or us3.
- Europe: use region-code=eu.
- Canada: use region-code=ca.
- Australia: use region-code=au.
- Japan: use region-code=ap.
```

### Get a Rapid7 AppSec API Key

> * Create an account in Rapid7.
> * Click on Settings → API Keys.
> * Generate your API key.

Reviewer: ```suggestion
1. Create an account in Rapid7.
2. Click **Settings** > **API Keys**.
3. Generate your API key.
```
